article_1.php

```diff
@@ -0,0 +1,132 @@
+                    <div class="article__text_tittle">Когда выйдет игра?</div>
+                    Мы пока не объявляли дату релиза. Подписывайтесь на наши соцсети, чтобы следить за ходом разработки.
+                    <div class="article__text_tittle">Будет ли графика, как в трейлере?</div>
+                    Трейлер отображает уровень графики и атмосферы, который мы стремимся реализовать в финальной версии игры.
+                    <div class="article__text_tittle">Нужно ли играть в предыдущие игры серии перед тем, как начать S.T.A.L.K.E.R. 2?</div>
+                    S.T.A.L.K.E.R. 2 — самостоятельная игра, с которой можно начать знакомство со вселенной. Для максимального погружения мы рекомендуем пройти предыдущие части серии.
+                    <div class="article__text_tittle">Какие локации мы наверняка сможем увидеть в игре?</div>
+                    Как и прежде, действие игры будет проходить на территории чзо. Новые локации соседствуют с уже знакомыми.
+                    <div class="article__text_tittle">Что за аномалию нам показали в трейлере?</div>
+                    Это новая аномалия, особенности которой сможете узнать в игре.
+                    <div class="article__text_tittle">Будут ли в игре новые артефакты и монстры?</div>
+                    Да. Уверены, что новые порождения зоны не оставят вас равнодушными.
+                    <div class="article__text_tittle">Кого нам показали в конце трейлера, это главный герой?</div>
+                    Это герой зоны — сталкер.
+                    <div class="article__text_tittle">В трейлере был показан геймплей?</div>
+                    Ролик создан на движке игры с использованием наших игровых моделей.
+                    <div class="article__text_tittle">Будет ли возможность создавать модификации?</div>
+                    Мы сделаем всё возможное, чтобы у моддеров было пространство для творчества сразу после выхода игры.
+                    <div class="article__text_tittle">Какие технологии используются для создания S.T.A.L.K.E.R. 2?</div>
+                    Визуальная составляющая — важнейшая часть погружения в мир S.T.A.L.K.E.R. 2, поэтому мы стремимся использовать самые передовые технологии. Игра создается на unreal engine, в производстве задействованы захват движений и лиц, а также фотограмметрия.
+                </div>
+                <!--Article Text-->
+            </div>
+            <!--Center Part-->
+
+            <div class="bottom__part">
+                <div class="write__comments">
+                    <div class="upper">
+                        <img src="../images/user.png" class="user__img">
+                        <textarea id="textarea__comment" placeholder="Оставьте комментарий"></textarea>
+                    </div>
+                    <div class="under">
+                        <input type="submit" class="comment__button">
+                    </div>
+                </div>
+
+                <div class="written__comments">
```

Comment:
Use for loop and list to generate comments

Reply:
yes sir